services/QuillLMS/db/migrate/20240521201204_add_project_to_evidence_automl_models.evidence.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# This migration comes from evidence (originally 20240521200827)
+class AddProjectToEvidenceAutomlModels < ActiveRecord::Migration[7.0]
+  def up
+    add_column :evidence_automl_models, :project, :string, null: true
+
+    # Backfill the location column with the original location
+    Evidence::AutomlModel.update_all(project: 'comprehension')
+
+    change_column_null :evidence_automl_models, :project, false
```

Comment:
Oh this is a nice method for migrations like this. Thanks for introducing me to it.